src/unlimited.cpp

```diff
@@ -210,12 +209,12 @@ void HandleExpeditedRequest(CDataStream& vRecv,CNode* pfrom)
             if (xpeditedBlk.size() < maxExpedited )
 	      {
 		LogPrint("blk", "Starting expedited blocks to peer %s (%d).\n", pfrom->addrName.c_str(),pfrom->id);
-		std::vector<CNode*>::iterator it = std::find(xpeditedBlk.begin(), xpeditedBlk.end(),((CNode*)NULL));
+		std::vector<CNode*>::iterator it = std::find(xpeditedBlk.begin(), xpeditedBlk.end(),(CNode*)NULL);  // find an empty array location
```

Comment:
I did, and had to add back because "release" branch is not c++11.  This is why this line is changed at all, I'll revert but leave the comment above as deadalnix suggested